Gruntfile.concurrent.js

```diff
@@ -15,7 +15,7 @@ module.exports = function gruntConfig(grunt) {
   const config = {
     env: {
       default: {
-        src: '.env.default'
+        src: '.env.defaults'
```

Comment:
I would keep this named "default". I see where you're going, but it's the _default env file_.

Reply:
`dotenv-webpack` plugin looks for `.env.defaults` hence the rename
I could get rid of the plugin, I added it for when I was debugging

Reply:
Can you remove the plugin if you are no longer using? I think it would be great to get these changes in to fix those tests.